content/en/blog/ospo-definition.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Open Source Program Office (OSPO) Definition"
+author: todogroup
+date: 2020-06-29
+---
+
+The TODO Group are [over 5 years old](https://todogroup.org/blog/welcome/) and as an organization have brought together the brightest in the field of open source program management. We have produced a variety of open source program management [guides](https://todogroup.org/guides/), [tools](https://github.com/todogroup/repolinter), [surveys](https://github.com/todogroup/survey) and even an [awesome list](https://github.com/todogroup/awesome-oss-mgmt) to advance the practice.
+
+Today, we are formalizing our [definition](https://github.com/todogroup/ospodefinition.org) of what an open source program (colloquially an OSPO) is based on our years of experience and to ensure we have a common lexicon in the industry when we describe open source programs:
+
+*An open source program office (OSPO) is designed to be the center of gravity for an organization's open source operations and structure. This can include training developers, ensuring legal compliance, engaging with and building communities, and defining policies that govern code usage, distribution, selection, auditing and more.*
```

Comment:
Please remove "designed to be". Not only is "is designed" a passive verb, it's also wibbly wobbly. Be definitive in a definition. An OSPO **is** the center of gravity…

Reply:
See updated version